### ISSUE TITLE
Add simple way to translate

### ISSUE THREAD

**Issue by cao-** [forrestguice/SuntimesWidget#248]
I'd like to translate this app into my native language, that is Italian.  But I don't know how to proceed.  On other projects there is usually a simple way of doing it, through for instance the Weblate or Stringlate platforms.  Can you add a similar way to this project too?  Or otherwise, can you kindly tell me how to submit a translation?

**Comment by forrestguice** [comment]
Hi. A translation into Italian would be much appreciated. Thanks for taking an interest.

The [contribution](https://github.com/forrestguice/SuntimesWidget/blob/master/CONTRIBUTING.md) page has some instructions for translation, but probably not as easy as the mentioned tools.

To summarize, you need to determine your locale code (I think its `it`) and the corresponding values folder (I think thats `values-it`), and put your translated values into that [strings.xml](https://github.com/forrestguice/SuntimesWidget/blob/master/app/src/main/res/values-it/strings.xml) file (app/src/main/res/values-it/strings.xml). The (default) english values are kept [here](https://github.com/forrestguice/SuntimesWidget/blob/master/app/src/main/res/values/strings.xml) (app/src/main/res/values/strings.xml). Fork the master branch, locate those files, then copy everything below the initial `default_location` stuff from the default into your file, and then translate each line (and remove any lines that have `translatable=false`). When you feel ready, submit a pull request and I'll review and merge.

I took a quick glance at Stringlate. I don't have any experience with it but I think its a viable option. If you use it, make sure you are translating the `master` branch (and not one of the numerous feature branches).  For example, I think pull #218 used Stringlate to make a minor correction, but since changes were against the wrong branch I wasn't able to cleanly merge them.

**Comment by forrestguice** [comment]
btw, I'm also available to help, but also preparing for a vacation. The plan is to go backpacking with family in a wilderness area in southern Utah (`Bears Ears`). I won't be responsive over the next week or two, but should be available after that.

**Comment by forrestguice** [comment]
This [guide](https://guides.github.com/activities/forking/) might be helpful too, if this is your first time making a pull request.

**Comment by cao-** [comment]
Thanks for the answers!  I've just submitted a pull request via Stringlate.  I hope I did everything right;  if not, tell me.  I think I did a good job in translating, but I'll surely revise my translations after I have seen them actually in use in the app.  Sometimes it was difficult to guess a right translation just by looking at the string out of context.
Have a nice vacation!  Enjoy looking at the moon and the suntimes.
